scripts/run-client-docker.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+set -e
+
+export AWS_ACCESS_KEY_ID=test
+export AWS_SECRET_ACCESS_KEY=test
+export AWS_ENDPOINT_URL="http://localstack:4566"
+export AWS_ACCESS_KEY_ID=test
+export AWS_SECRET_ACCESS_KEY=test
+export AWS_REGION=us-east-1
+
+
+for i in 0 1 2
+do
+  echo "Checking hawk\_participant\_${i} at <url>:300${i}/health..."
+  curl -f hawk_participant_${i}:300${i}/health
+done
+
+echo "All endpoints are healthy."
+
+
+#/bin/client \
+#    --request-topic-arn arn:aws:sns:$AWS_REGION:000000000000:iris-mpc-input.fifo \
```

Comment:
Once we have a functioning server, able to process requests, we can re-enable this

Reply:
When this command succeeds then the test is complete, correct?

Reply:
Yup. Locally we should be able to verify this by running the 3 services, and issuing this command. The server should be able to process all the SQS messages being generated.